extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/cors/CORSConfig.java

```diff
@@ -19,7 +19,7 @@
      *
      * default: returns any requested origin as valid
      */
-    @ConfigItem
+    @ConfigItem(defaultValue = "*")
```

Comment:
I'm not sure it should be a default value - that would mean all other properties should also be defaulted to `*`

Reply:
We should probably revert this and allowsAllOrigin if the list contains only `*` or `isEmpty()`

Reply:
@machi1990 yes, I agree

Reply:
We may need to allow `*` for more of the cors properties anyway in the future.
`Access-Control-Expose-Headers`, `Access-Control-Allow-Methods` and `Access-Control-Allow-Headers` allow * as wildcard value.

Reply:
@Postremus sure, it makes sense for the consistency purposes. But lets keep the current pattern, it is either absent or set to `*`